app/src/main/java/de/danoeh/antennapod/fragment/preferences/NetworkPreferencesFragment.java

```diff
@@ -174,4 +152,145 @@ private void showUpdateIntervalTimePreferencesDialog() {
         }
         return entries;
     }
+
+    private void FeedRefreshIntervalDialog()
+    {
+        final int[] string_spinnercase = new int[1];
+        final Context context = getActivity();
+        AlertDialog.Builder builder = new AlertDialog.Builder(context);
+        builder.setTitle(R.string.pref_autoUpdateIntervallOrTime_titleAct);
+        builder.setMessage(R.string.pref_autoUpdateIntervallOrTime_sumAct);
+        LayoutInflater inflater = requireActivity().getLayoutInflater();
+
+        View v_iew=inflater.inflate(R.layout.update_refresh_time_dialoxg, null);
+        builder.setView(v_iew);
+
+        Spinner spinner = (Spinner) v_iew.findViewById(R.id.spinner);
+        RadioButton radioButton_interval = (RadioButton) v_iew.findViewById(R.id.intervalRadioButton);
+        RadioButton radioButtonTime = (RadioButton) v_iew.findViewById(R.id.timeRadioButton);
+        RadioButton radioButton_disable = (RadioButton) v_iew.findViewById(R.id.disableButton);
+        RadioGroup radioGroupOne = (RadioGroup) v_iew.findViewById(R.id.genderRadioGroup);
+        TimePicker timePicker = (TimePicker) v_iew.findViewById(R.id.timePicker);
```

Comment:
Please use 24 hour mode instead of am/pm if the system is set to 24 hour mode. I think something like this could work:

    picker.setIs24HourView(DateFormat.is24HourFormat(getContext()));